### PR TITLE
Dont touch it xd

### PR DESCRIPTION
**Pull request by mwawrzen**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,28 +1,122 @@
...
-    <h1>polecat-ferret</h1>
-
-    <label class="switch">
-        <input type="checkbox" onclick="mode()" />
-        <span class="slider"></span>
-    </label>
-
-    <script src="js/modeSwitch.js"></script>
-
+    <div class="corner">
```
Comment (jaanonim) [inline src/index.html]: may be insted of div asside?

File: src/index.html [hunk 2]
```diff
@@ -1,28 +1,122 @@
...
+    </section>
+    <section class="page" id="page-2">
+        <header class="page__title">Technik<br>teleinformatyk</header>
+        <p class="page__subtitle">to jest kierunek w naszej szkole</p>
+        <button
+            type="button"
+            class="page__button btn-2"
+            onclick="readMore()"
+        >
+            Czytaj więcej..
```
Comment (jaanonim) [inline src/index.html]: Mising dot

File: src/css/main.sass [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        top: 30px
+        left: 30px
+        width: 50px
+        height: 50px
+        display: grid
+        place-items: center
+        transition: .2s
+
+        &:hover
+            transform: rotate(180deg)
```
Comment (jaanonim) [inline src/css/main.sass]: 90deg will look better

**Review by jaanonim — CHANGES_REQUESTED** [review]
Some minor changes to make

**Review by n2oneProgrammer — COMMENTED** [review]
LGTM

**Pull request merged**